src/iuse.cpp

```diff
@@ -8074,27 +8059,57 @@ int iuse::autoclave( player *p, item *it, bool t, const tripoint &pos )
             return 0;
         }
 
-        auto reqs = *requirement_id( "autoclave_item" );
-
-        item_location to_sterile = game_menus::inv::sterilize_cbm( *p );
+        bool empty = true;
+        item *clean_cbm = nullptr;
+        for( item &bio : it->contents ) {
+            if( bio.is_bionic() ) {
+                clean_cbm = &bio;
+            }
+        }
+        if( clean_cbm ) {
+            empty = false;
+            if( query_yn( _( "Autoclave already contains a CBM.  Do you want to remove it?" ) ) ) {
+                g->m.add_item( pos, *clean_cbm );
+                it->remove_item( *clean_cbm );
+                if( !query_yn( _( "Do you want to use the autoclave?" ) ) ) {
+                    return 0;
+                }
+                empty = true;
+            }
+        }
 
-        if( !to_sterile ) {
+        // Power_draw seem to consume random amount of battery so +100 to be safe
+        if( ( ( it->type->tool->power_draw / 1000 ) * to_seconds<int>( 90_minutes ) ) / 1000 + 100 >
```

Comment:
This is a long term. It's difficult to see that this is the same term as used below in the popup and it would be much better if you stored this in a variable so the term only appears once.

Reply:
Better?